plugins/woocommerce-admin/package.json

```diff
@@ -10,7 +10,7 @@
 	},
 	"scripts": {
 		"changelog": "pnpm --filter=woocommerce -r run changelog ",
-		"turbo:build": "pnpm run clean && cross-env NODE_ENV=production WC_ADMIN_PHASE=core webpack",
+		"turbo:build": "node ./bin/replace-editsite-lock-unlock.js && pnpm run clean && cross-env NODE_ENV=production WC_ADMIN_PHASE=core webpack",
```

Comment:
Nitpick: It my be cleaner to run this as a postinstall script, so it only needs to be run once:

```
"postinstall": "node ./bin/replace-editsite-lock-unlock.js"
```

Reply:
I wrote this before seeing the webpack solution -- feel free to ignore it 😄

Reply:
Nice! Thank you @chihsuan @adrianduffell for suggestions 👍 I'll try webpack approach and update the code.

Reply:
Now that webpack resolves `lock-unlock.js` to the local script, you can remove calls to the file in npm scripts, right?

Reply:
@psealock You're right. I forgot to remove them 🤦 Updated in https://github.com/woocommerce/woocommerce/pull/40884/commits/2b9fe30ecc24f46ef8504705e8804d288b2690a0